xbmc/GUIPassword.cpp

```diff
@@ -29,6 +30,7 @@
 #include "utils/StringUtils.h"
 #include "utils/Variant.h"
 #include "view/ViewStateSettings.h"
+#include "utils/log.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers